packages/react/menu/src/Menu.tsx

```diff
@@ -73,7 +227,7 @@ const [MenuContentProvider, useMenuContentContext] = createContext<MenuContentCo
 );
 
 type MenuContentOwnProps = Polymorphic.Merge<
-  Polymorphic.OwnProps<typeof MenuContentImpl>,
+  Polymorphic.OwnProps<typeof MenuRootContent>,
```

Comment:
We purposefully don't union `MenuRootContent` and `MenuSubContent` props here because they have conflicting prop types. We discussed previously that we would just allow the `MenuSubContent` to accept props that it would ignore.

Reply:
Should we add this as a comment in the code for our future selves?